### DISCUSSION TITLE
[Vue warn]: Failed to resolve component: q-btn(and all others components)

### DISCUSSION THREAD

**Discussion by Venezyz** [quasarframework/quasar#12550] [General - Components / Directives / etc]
hi guys... 
I created a vue3 project, than add Quasar. Than I past a component file from Quasar.dev called "Quasar Layout Header and Footer" (https://quasar.dev/layout/header-and-footer) but does not render... 
Someone could help? tks.

**Comment by MilosPaunovic** [comment]
You'll need to supply some more details in order for someone to be able to help; version of Quasar used, way you've added it to your project... As much as you can.

**Reply by Venezyz** [reply]
First I created a project "vue create my-project". Then "npm install quasar". 
The package.json I pasted below.

So I created some pages with vue3, pure CSS and it worked normally. But when I copied "Quasar Layout Header and Footer" (https://quasar.dev/layout/header-and-footer) any Quasar class ("q-btn" or "q-layout" for example) works. 

I inspected it and it showed yellow alerts "[Vue warn]: Failed to resolve component: q-btn"  and all others Quasar class component (q-layout, q-cards etc...)

/*package.json*/
{
  "name": "front",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build"
  },
  "dependencies": {
    "@quasar/extras": "^1.0.0",
    "animate.css": "^4.1.1",
    "core-js": "^3.8.3",
    "quasar": "^2.0.0",
    "register-service-worker": "^1.7.2",
    "vue": "^3.2.13",
    "vue-router": "^4.0.3",
    "vuex": "^4.0.0"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "~5.0.0",
    "@vue/cli-plugin-pwa": "~5.0.0",
    "@vue/cli-plugin-router": "~5.0.0",
    "@vue/cli-plugin-vuex": "~5.0.0",
    "@vue/cli-service": "~5.0.0",
    "vue-cli-plugin-quasar": "~4.0.4"
  }
}

**Comment by MilosPaunovic** [comment]
If you just installed `Quasar` with `npm i quasar` command, you'll need to tell your `Vue` app to use it. Better solution would be to use [vue-cli-plugin-quasar](https://github.com/quasarframework/vue-cli-plugin-quasar#getting-started) which does that for you and even better one would be to use `Quasar CLI`.

**Comment by Venezyz** [comment]
well... I user a Quasar Cli to re-create a project, using "Quasar create App" command... And this time works fine...
I don´t know why "vue create app" don´t work for me...

**Comment by linrongbin16** [comment]
same issue, I use vue/cli@5.0.4 (latest official version), and I don't want to use quasar/cli, still stick to vue/cli, vue/cli@5.0.0-rc1 is working good with quasar.
